nixos/modules/services/web-apps/plausible.nix

```diff
@@ -248,6 +248,7 @@ in {
             # setup
             ${cfg.package}/createdb.sh
             ${cfg.package}/migrate.sh
+            ${cfg.package}/bin/plausible eval '(Plausible.Release.prepare() ; Plausible.Auth.create_user("$ADMIN_USER_NAME", "$ADMIN_USER_EMAIL", "$ADMIN_USER_PWD"))'
```

Comment:
this is not working, because of single quote in `eval`

Reply:
@kirillrdy where? do you mean I shouldn't be using single quotes at all? Or is there a misplaced one I can't see and am blind to? Thanks.

Reply:
this worked last time i checked
`${cfg.package}/bin/plausible eval "(Plausible.Release.prepare() ; Plausible.Auth.create_user(\"$ADMIN_USER_NAME\", \"$ADMIN_USER_EMAIL\", \"$ADMIN_USER_PWD\"))"`

single quotes passed to bash will mean it will not expand variables

Reply:
> single quotes passed to bash will mean it will not expand variables

Oh got it! You are absolutely right! Forgot about that and this is bash haha Thanks as always.

Reply:
all working 🎉